### PR TITLE
Add doc page for cloud-init daily builds

### PR DESCRIPTION
**Pull request by TheRealFalcon**

### CODE REVIEW

File: doc/cloud-init-daily-builds.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+```bash
+new_upstream_snapshot.py
+# Tool fails to apply a patch
+quilt push -f
+# Manually fix the patch, then run the next command
+quilt refresh
+# The following will ensure that the patches all apply and tests pass
+quilt pop -a
+tox
+quilt push -a
```
Comment (holmanb) [inline doc/cloud-init-daily-builds.md:64]: push then tox then pop?

**Review by holmanb — APPROVED** [review]
One nit, otherwise looks good!

File: doc/cloud-init-daily-builds.md (hunk 1 above)
Comment (TheRealFalcon) [reply doc/cloud-init-daily-builds.md:64]: Yeah, tests should pass with patches applied. Should I add a `-e py3` or is there another concern.

Comment (holmanb) [reply doc/cloud-init-daily-builds.md:64]: I was trying to point out that the instructions look wrong - as written it is pop then tox then push.

Comment (TheRealFalcon) [reply doc/cloud-init-daily-builds.md:64]: Oh! Yeah... I should fix that 🤦‍♂️

**Review by aciba90 — APPROVED** [review]

**Pull request merged**